cli/pcluster/cli.py

```diff
@@ -334,6 +342,11 @@ def _get_parser():
         help="Specifies the cookbook to use to build the AWS ParallelCluster AMI.",
     )
     _addarg_config(pami)
+    pami.add_argument(
+        "-t",
+        "--cluster-template",
+        help="Specifies the cluster section of the ParallelCluster configuration file to retrieve VPC settings."
```

Comment:
"AWS ParallelCluster" or "Specifies the cluster section of the configuration file to retrieve VPC settings". Same for the other help string

Reply:
I don't understand this. Why specify the cluster section when the vpc section is a different section?

Reply:
Because we are using the ParallelCluster config parser behind the scenes, that searches for cluster section (Note: currently the `pcluster createami` command is searching for the `[cluster default]` section).

An alternative could be to pass the vpc id and the subnet id directly to the createami command.

Reply:
I'd opt for the latter, I know it's more work but it's a much better user experience

Reply:
I added both the possibilities